salt/utils/__init__.py

```diff
@@ -13,6 +13,7 @@
 
 # Import Salt libs
 from salt.defaults import DEFAULT_TARGET_DELIM
+from salt.utils.dependencies import depends
```

Comment:
You're suggesting we should import depends from `salt.utils`.
Besides namespacing, the reason why we now have `salt.utils.files` and the like is to improve import time speed.
This would go against that work.

Reply:
I think this will be used often enough (450+ times across the code base) to warrant allowing this import.

Reply:
Sure, often enough, but it still doesn't remove the fact that it will trigger a bunch of other imports which might not matter for most of those, 450+ occurrences.

Reply:
The only other thing its causing to be imported is `salt.utils.versions.LooseVersion`

Reply:
I moved this from `salt.utils.depends` to `salt.version.reqs`